README.md

```diff
@@ -1 +1,58 @@
-# hello_microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+## Description
+
+![hello_microverse snapshot](./images/hello_microverse.png)
+
```

Comment:
- [ ] OPTIONAL: It would be better if you add another sub-title `## Screenshot` for the project screenshot instead of inserting it in the `Description` part. Please do that for a better Readme design.